### ISSUE TITLE
Duplicate Entries for መልክአ፡ ሚካኤል፡ - LIT1860Malkea & LIT2835RepCh119

### ISSUE THREAD

**Issue by smaugustine** [BetaMasaheft/Documentation#1505]
This is a more hefty duplicate situation, 14 MSS vs 44 MSS. Shall I begin changing the refs over?

**Comment by eu-genia** [comment]
Yes, LIT1860Malkea should be deleted and merged into LIT2835RepCh119.
Thank you @smaugustine for taking care of this too.
(Some history: It was decided at one point to go after Chaîne. But when we (Massimo) created new IDs for all Chaîne items it often was not checked where the work records already existed (sometimes old records were deleted and replaced, but often not). As a result, doublets emerged. And the titles assigned to the Chaîne items (Hymn to Michael, Hymn to Mary, Hymn to Jesus etc) are often less telling than Malke'a ... and so on, so the "old" records were kept being used instead of the "new" ones.)

**Comment by DenisNosnitsin1970** [comment]
Please transfer incipit /explicit into the record that will remain, and you can add one more stanza... I must say that IDs based on pure numbers + the number in Chaine create difficulties for some people. Being purely numerical is not wrong, but it is very difficult to work only with numbers, without the slightest reference to the content, and it is more difficult to find purely numerical IDs, esp. if there are no reasonable incipits.

**Comment by eu-genia** [comment]
While we are at it I wonder if LIT1858Malkea (used in 1 MS see https://betamasaheft.eu/iiif/EMIP/Codices/37/148.tif/full/full/0/default.jpg) is also one of the RepCh-doublets

**Comment by DenisNosnitsin1970** [comment]
LIT2891RepCh172_

**Comment by DenisNosnitsin1970** [comment]
?

**Comment by eu-genia** [comment]
sorry I have just seen that this branch was never pushed and the references probably never updated either, the records may be partially outdated, @thea-m can you check if there is more to be done? thank you
